pipeline/stages/test.yaml

```diff
@@ -0,0 +1,157 @@
+parameters:
+  - name: env
+    type: string
+    default: 'sbox'
+
+stages:
+  - stage: Test${{parameters.env}}
+    displayName: 'Test to ${{parameters.env}}'
+    dependsOn: Apply${{parameters.env}}
+    variables:
+      - template: ../variables/variables-common.yaml
+      - template: ../variables/variables-${{parameters.env}}.yaml
+      - group: cvp-${{parameters.env}}
+    jobs:
+      - job: LoadBalancerTest
+        displayName: '${{parameters.env}} Load Balancer Tests '
+        steps:
+          - task: AzureCLI@2
+            name: 'healthprodcheck'
+            displayName: Check Health Probe
+            inputs:
+              azureSubscription: '${{variables.subscriptionName}}'
+              scriptType: 'bash'
+              scriptLocation: 'inlineScript'
+              inlineScript: |
+                service_name="${{variables.product}}-recordings-${{parameters.env}}-"
+                rgName="${service_name}rg"
+                lbName="${service_name}lb"
+
+                metricName="DipAvailability"
+                resourceType="Microsoft.Network/loadBalancers"
+
+                sleep 2m
```

Comment:
I'm probably missing something obvious but what's with the 2min sleep?

Reply:
The Metrics request returns the avg every 1 minute, so leaving it 2 minutes to capture some results before checking it out

Reply:
ahhhh, so yes, I was missing something obvious :D